settings.py

```diff
@@ -3,8 +3,8 @@
 OTHER_PARAMS = '&sign=&expire='
 SERVICE_ID = '808'
 
-PHONE_FIRST = '' # your phone number
-QUEUE_SITE = ''  # your queue url (site where to monitor queue)
+PHONE_FIRST = '' # your Polish phone number, starting with +48
+QUEUE_SITE = ''  # your queue url (site where to monitor queue, e.g. https://eo.pasport.org.ua/index.php)
```

Comment:
This is universal tool for the monitoring of any queue, therefore specifying any concrete links is not necessary here.

Reply:
it's just an example; I personally entered `https://wroclaw.pasport.org.ua/solutions/e-queue` at first 😄 
maybe we need to clarify somehow that this site should be extracted from the API